### PR TITLE
Fix: File upload column update

### PR DESCRIPTION
**Pull request by gmn0105**

### CODE REVIEW

File: db/samplev2.py [hunk 1]
```diff
@@ -60,14 +60,18 @@ def generate_sample_db(path, course_id, course_name, db_session):
             db_session.rollback()
 
         # Add scores of the student
+        #If you want the entries to be displayed in order in DB Browser then use this otherwise comment it.
+        if(db_session.query(Score).filter_by(student_id = formatted_student_id).first() != None):
+            db_session.query(Score).filter_by(student_id = formatted_student_id, course_id = course_id).delete(synchronize_session = 'fetch')
```
Comment (shreyasnbhat) [inline db/samplev2.py]: Why is this `synchronize_session = 'fetch'` needed?

Comment (shreyasnbhat) [reply db/samplev2.py]: The 'fetch' strategy results in an additional SELECT statement emitted and will significantly reduce performance.

Comment (gmn0105) [reply db/samplev2.py]: False - don’t synchronize the session. This option is the most efficient and is reliable once the session is expired, which typically occurs after a commit(), or explicitly using expire_all(). Before the expiration, objects may still remain in the session which were in fact deleted which can lead to confusing results if they are accessed via get() or already loaded collections.

'fetch' - performs a select query before the delete to find objects that are matched by the delete query and need to be removed from the session. Matched objects are removed from the session.

I figured fetch was the correct option for our purpose..

Comment (gmn0105) [reply db/samplev2.py]: Actually evaluate will also serve the purpose.. Ill update it

File: db/samplev2.py [hunk 2]
```diff
@@ -62,7 +62,7 @@ def generate_sample_db(path, course_id, course_name, db_session):
         # Add scores of the student
         #If you want the entries to be displayed in order in DB Browser then use this otherwise comment it.
         if(db_session.query(Score).filter_by(student_id = formatted_student_id).first() != None):
-            db_session.query(Score).filter_by(student_id = formatted_student_id, course_id = course_id).delete(synchronize_session = 'fetch')
+            db_session.query(Score).filter_by(student_id = formatted_student_id, course_id = course_id).delete(synchronize_session = 'evaluate')
```
Comment (shreyasnbhat) [inline db/samplev2.py]: Why won't synchronize_session = False with db_session.expire() work?

File: db/samplev2.py [hunk 3]
```diff
@@ -62,7 +62,7 @@ def generate_sample_db(path, course_id, course_name, db_session):
         # Add scores of the student
         #If you want the entries to be displayed in order in DB Browser then use this otherwise comment it.
         if(db_session.query(Score).filter_by(student_id = formatted_student_id).first() != None):
```
Comment (shreyasnbhat) [inline db/samplev2.py]: `!=` convert to `is not`

**Pull request closed without merging**